PM2023-001 - Total node failure.md

```diff
@@ -0,0 +1,63 @@
+| 2023-02-14 10:48 | We have no nodes                                                                                                                                                                                                                                                                                               |
+| 2023-02-14 10:49 | Attempt at running node rollover script to add new nodes                                                                                                                                                                                                                                                       |
+| 2023-02-14 10:51 | Attempt fails due to timeout from Docker Hub                                                                                                                                                                                                                                                                   |
+| 2023-02-14 10:52 | 2nd attempt at running node rollover script to add new nodes                                                                                                                                                                                                                                                   |
+| 2023-02-14 10:57 | New nodes are being added                                                                                                                                                                                                                                                                                      |
+| 2023-02-14 11:01 | LB detects new nodes as healthy, allowing traffic                                                                                                                                                                                                                                                              |
+| 2023-02-14 11:07 | Sev 1 ticket raised                                                                                                                                                                                                                                                                                            |
+| 2023-02-14 11:59 | Sev 1 ticket marked as resolved                                                                                                                                                                                                                                                                                |
+
+
+## Contributing Factors
+
+- QA config differs from prod, missing staging environment
+- QA test suite doesn't cover fluentd log collection
+- Rollover process has no documentation in place for what happens if it goes bad
+- During the attempt to revert the container-runtime, the value was thought to be dockershim, but was instead dockerd
```

Comment:
Could such toggle-values be validated in some way, as if they were enums?
We need to be careful that major changes such as the containerd change doesn't accidentially slip through

Reply:
@emil2k @avnes

Reply:
Validation of possible container runtime values has been added here: https://github.com/dfds/infrastructure-modules/pull/678